clang/lib/Analysis/FlowSensitive/TypeErasedDataflowAnalysis.cpp

```diff
@@ -221,18 +221,21 @@ class PrettyStackTraceCFGElement : public llvm::PrettyStackTraceEntry {
 // Avoids unneccesary copies of the environment.
 class JoinedStateBuilder {
   AnalysisContext &AC;
+  Environment::ExprJoinBehavior ExprBehavior;
```

Comment:
If you're going to shorten the variable name, then I think JoinBehavior might be better (since this doesn't have to do with the behavior of expressions).

Reply:
Agreed -- done! (I shortened the name because I didn't want to reuse the type name `ExprJoinBehavior` as a variable name -- true, the type is scoped to `Environment`, but this still seemed potentially confusing.)